app/master/build.py

```diff
@@ -471,19 +473,21 @@ def _perform_async_postbuild_tasks(self):
         data. This method also transitions the FSM to finished after the postbuild tasks are complete.
         """
         try:
-            self._create_build_artifact()
+            timing_data = self._read_subjob_timings_from_results()
+            self._create_build_artifact(timing_data)
```

Comment:
It seems like there's not much point to moving the `self._read_subjob_timings_from_results()` call out of `self._create_build_artifact()`.